contracts/voting_escrow/README.md

```diff
@@ -9,99 +9,74 @@ gains 250 voting power. Voting power is linearly decreased by passed periods. On
 ```json
 {
   "owner": "terra...",
-  "anchor_token": "terra..."
+  "anchor_token": "terra...",
+  "min_lock_time": 2419200,
+  "max_lock_time": 62899200,
+  "period_duration": 604800,
+  "boost_coefficient": 25
 }
 ```
 
 ## ExecuteMsg
 
-### `receive`
+### `extend_lock_amount`
 
-Create new lock, extend current lock's amount or deposit on behalf other address.
+Extends lock amount for specified user.
 
 ```json
 {
-  "receive": {
-    "sender": "terra...",
-    "amount": "123",
-    "msg": "<base64_encoded_json_string>"
+  "extend_lock_amount": {
+    "user": "terra...",
+    "amount": 500
```

Comment:
500 here should be "500" because the type of amount is Uint128.

Reply:
addressed 👍